docs/reference/verify-assets.mdx

```diff
@@ -160,63 +219,70 @@ WARN[0000] image signature verification is disabled due to using corresponding o
 WARN[0000] image signature verification is disabled due to using corresponding option
 RUNTIME.CONTAINERNAME  PID          UID          GID          MNTNS_ID RET FL… MODE        COMM        FNAME                  TIMESTAMP
 ```
+</TabItem>
 
-### With `kubectl-gadget`
+<TabItem value="ig-daemon" label="ig daemon">
 
-Compared to `ig`, you cannot specify information with regard to verification when calling `kubectl gadget run`:
+Like `kubectl-gadget`, you cannot specify information with regard to verification when calling `gadgetctl run`:
 
 ```bash
-$ kubectl gadget run --public-keys="$(cat your-key.pub)" trace_exec
+$ gadgetctl run --public-keys="$(cat your-key.pub)" trace_exec --remote-address=$addr
 Error: unknown flag: --public-keys
-$ kubectl gadget run --verify-image=false trace_exec
+$ gadgetctl run --verify-image=false trace_exec --remote-address=$addr
 Error: unknown flag: --verify-image
 ```
```

Comment:
Similar to above, I don't think we should give examples that just result in errors like `unknown flag` - those flags  might still be remembered as valid, because it looks like a valid example at first. Explanation alone should suffice, IMHO.